example/lambdas.yml

```diff
@@ -6,6 +6,7 @@ CNMToCMA:
   s3Source:
     bucket: '{{buckets.shared.name}}'
     key: daacs/podaac/cnmToGranule-1532625517.zip
+    uniqueIdentifier: '1532625517'
```

Comment:
If `uniqueIdentifier` is not provided what happens? Should one be generated?

Reply:
Great question - If uniqueIdentifier isn't provided, the kes override method that's injecting the references in place of the user provided arn will *not* be replaced see `lookupLambdaReference` in kes.js , and no references will be generated in WorkflowLambdaVersions.